src/Nethermind/Nethermind.JsonRpc/Modules/Admin/AdminRpcModule.cs

```diff
@@ -139,4 +143,19 @@ public ResultWrapper<PruningStatus> admin_prune()
     {
         return ResultWrapper<PruningStatus>.Success(_pruningTrigger.Trigger());
     }
+
+    public ResultWrapper<string> admin_verifyTrie()
+    {
+        if (_blockTree.Head is null)
+        {
+            return ResultWrapper<string>.Fail("Head is null. Unable to know state root to verify.");
```

Comment:
Potentially can we pass the `BlockParameter` where we want to verify tries?
Also potentially should we add `admin_availableStateRoots`, where it would look-up where state root is available in some block range?

Reply:
Sure, why not.